composer/loggers/console_logger.py

```diff
@@ -203,6 +210,8 @@ def log_to_console(self, data: Dict[str, Any], state: State, prefix: str = '', i
         for data_name, data in data.items():
             data_str = format_log_data_value(data)
             log_str += f'\n\t {prefix}{data_name}: {data_str}'
+        for table_name, table in self.tables.items():
+            log_str += f'\n\t {prefix}{table_name}: {table}'
```

Comment:
can you make a jira to make this prettier? not priority, just would be nice if it printed an actual table

Reply:
https://mosaicml.atlassian.net/browse/GRT-2303